rllib/evaluate.py

```diff
@@ -1,10 +1,9 @@
 #!/usr/bin/env python
 
-import argparse
+import json
```

Comment:
Not: Sort alphabetically. :)

Reply:
@sven1977 if I understand correctly, you wanted me to sort the non-ray imports alphabetically. if so, that's done.